Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex

```diff
@@ -137,15 +141,30 @@ \section{Persistent data management}
 Moodcat has to handle both song data and chatroom data.
 
 \par
-For the song we store the basic properties, like artist, song title, genre and duration.
-Furthermore each song has its own \gls{valence} and \gls{arousal} value, which is computed and corrected by the music matching algorithm.
-The algorithm uses some of the audio features of the track for the computation of the valence and arousal values, these features are stored with the song.
+For the artist we only store our own artist id and artist name.
+
+\par
+For the song we store the basic properties, like artist, song title and duration.
+A song has an unique id in our system and also an id from \gls{Soundcloud}, so we can fetch data from Soundcloud.
+We also include an url to display the artwork of a song.
+
+Furthermore each song has its own \gls{valence} and \gls{arousal} value, this is updated when users classify the song.
+This classification will be done in the Music Matching Algorithm as described above. So, we also have keep count of relative amount of up-votes compared to the down-votes.
+
+\par 
+For the room we store our roomid. 
+We also store the basic properties, roomname,the currently playing song and the duration.
+Furthermore, the arousal and valence values of the room are stored.
+
+\par
+For the user we only store the id and the username of the user. 
```

Comment:
Currently we don't store this, but we want to store this in the future.
Would you delete this sentence?